data/src/main/java/com/bottlerocketstudios/brarchitecture/data/model/RepositoryDto.kt

```diff
@@ -8,11 +8,11 @@ import java.time.ZonedDateTime
 
 @JsonClass(generateAdapter = true)
 @Parcelize
-data class Repository(
+data class RepositoryDto(
```

Comment:
@VanyaHuamanBottleRocket Refactor to `GitRepositoryDto`

Reply:
That seems like the better option.